modules/rosa-setting-up-cli.adoc

```diff
@@ -0,0 +1,46 @@
+:_module-type: PROCEDURE
+// Module included in the following assemblies:
+//
+// * cli_reference/rosa_cli/rosa-get-started-cli.adoc
+
+
+[id="rosa-setting-up-cli_{context}"]
+= Setting up the `rosa` CLI
+
+[role="_abstract"]
+To set up the `rosa` CLI, download the latest release, then configure and initialize `rosa`:
+
+.Procedure
+
+. Download the latest release of `rosa` for your operating system from the link:https://github.com/openshift/rosa/releases/latest[`rosa` releases page on GitHub].
++
+. (Optional) After downloading a release, you can rename the executable file you download to `rosa`, and then add `rosa` to your path.
```

Comment:
We generally use the ISG formatting for "Required" and "Optional", so:
s/(Optional) After/Optional: After

s/you download/you downloaded

Reply:
Done.